repose-aggregator/artifacts/docker/src/docker/resources/file/ubuntu/Dockerfile

```diff
@@ -0,0 +1,50 @@
+# Use Ubuntu as a base, since it works and is familiar.
+# The version should be the latest tested, LTS version.
+FROM ubuntu:16.04
+
+MAINTAINER The Repose Team <reposecore@rackspace.com>
+
+# Setting constants
+ENV APP_ROOT=/etc/repose
+ENV APP_VARS=/var/repose
+ENV APP_LOGS=/var/log/repose
+ENV PKG_DIR=/tmp/packages/repose/
+
+# This environment variable is used to set command-line options.
+# The user can manually set these options using the "-e" flag with the Docker run command.
+ENV JAVA_OPTS=
+
+# Copy local Repose packages
+COPY repose_*.deb ${PKG_DIR}
+COPY repose-filter-bundle_*.deb ${PKG_DIR}
+COPY repose-extensions-filter-bundle_*.deb ${PKG_DIR}
+COPY repose-experimental-filter-bundle_*.deb ${PKG_DIR}
```

Comment:
These `COPY`'s should probably move below the `RUN apt-get...` so that one more possibly cached point can be created/used.
This will prevent having to download and install OpenJDK every time.